### PR TITLE
update: fixes #335

### LINKED ISSUE #335: Receives [ERROR] Unknown arguments: -b when running ./docs/doxygen/generate_docs.bash
| | |
|:---|:---|
|**_F´ Version_**| 1.5>=|
|**_Affected Component_**|all   |
---
## Problem Description

Receives [ERROR] Unknown arguments: -b when running:
```./docs/doxygen/generate_docs.bash```

## How to Reproduce
run the following from fprime directory:
```./docs/doxygen/generate_docs.bash```

## Expected Behavior
Generate doxygen and sphinx documents

### PR DESCRIPTION
**Pull request by saba-ja**
## Change Description

Failing issue of generate_docs.bash when generating doxygen and sphinx is resolved. 

1. The script will use a temporary folder name with a prefix that is used by fprime-util build. 
2. Sphinx dependency packages are checked before attempting to run sphinx.

### CODE REVIEW

File: docs/doxygen/generate_docs.bash [hunk 1]
```diff
@@ -75,6 +75,7 @@ function clobber
 # Python
 (
     clobber "${PY_OUTPUT}"
+    pip install sphinx sphinx-rtd-theme sphinx-autoapi sphinx-autoapi recommonmark sphinxcontrib.mermaid
```
Comment (LeStarch) [inline docs/doxygen/generate_docs.bash:78]: Why is this line needed?  It seems like it should already be part of the VENV.

Comment (saba-ja) [reply docs/doxygen/generate_docs.bash:78]: I agree, they could be moved to 
https://github.com/nasa/fprime/blob/261ae546d93feb779536bee393caeb8c5140167d/Fw/Python/setup.py#L87

extras_require={"dev": ["black", "pylama", "pylint", "pre-commit"]},

**Comment by saba-ja** [comment]
Will re-submit the pull request with the modification specified above.
